test/services/checklist_network_services_test.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:checklist/constants.dart';
+import 'package:checklist/exceptions/custom_exceptions.dart';
+import 'package:checklist/models/checklist_item.dart';
+import 'package:checklist/repositories/auth_repository.dart';
+import 'package:checklist/services/checklist_network_services.dart';
+import 'package:dio/dio.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+class MockDioClient extends Mock implements Dio {}
```

Comment:
Use separate file for Mocked Classes which are being reused across the test files.